builtin/providers/circonus/descriptions.go

```diff
@@ -0,0 +1,29 @@
+package circonus
+
+var (
+	accountDescription   map[string]string
+	collectorDescription map[string]string
+	providerDescription  map[string]string
+)
+
+func init() {
+	// NOTE(sean@): needs to be completed
+	accountDescription = map[string]string{
+		accountContactGroupsAttr: "Contact Groups in this account",
+		accountInvitesAttr:       "Outstanding invites attached to the account",
+		accountUsageAttr:         "Account's usage limits",
+		accountUsersAttr:         "Users attached to this account",
+	}
+
+	// NOTE(sean@): needs to be completed
```

Comment:
There are missing descriptions that need to be populated, but I'm going to punt on this for now and let the vendor populate these values.  Most of our providers don't seem to have this value populated so I didn't consider it mandatory.  LMK if this is considered mandatory or not.